app/views/hire.erb

```diff
@@ -0,0 +1,4 @@
+<form action="/hire" method="post">
+  <input type="text" name="date" placeholder="date">
```

Comment:
how about a placeholder that is the required date format?
DD/MM/YYYY

Reply:
There is no required format, it's just stored as a string in the DB

Reply:
shall we do it after MVP confirmed?